src/components/LogIn/LogIn.js

```diff
@@ -243,19 +366,6 @@ function LogIn() {
                 backgroundColor: globalConstant.whiteColor,
               },
             ]}
-          >
-            <Text style={styles.textAG}>Sign In with Apple</Text>
-          </TouchableOpacity>
```

Comment:
why is this commented? if its for google we can remove it well change change the text and the function after